microscopium/data/default-settings.yaml

```diff
@@ -0,0 +1,64 @@
+global:
+  # where to put the output file. The rules applied to this setting are:
```

Comment:
If the rule for determining if it's an absolute or relative path is "does it start with a forward-slash or not?" this should also be in the comments.
Ideally there should also be a windows and unix style example "/c:/Users/..." & "/home/user...".

Reply:
That's not the rule. Python has a function to do this because it's more complicated than that. On windows, /c/ will work but so will C:\

Reply:
Ok then. 
My comment was to say that we should write extra detail if this parsing *isn't* completely foolproof to  users who know nothing about the implementation. If you think it's foolproof we can close this conversation, I trust your judgement